src/Wellcome.Dds/CatalogueClient/Program.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.IO;
+using System.Net.Http;
+using System.Text.Json;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Options;
+using Utils;
+using Wellcome.Dds.Catalogue;
+using Wellcome.Dds.Common;
+using Wellcome.Dds.Repositories.Catalogue;
+
+namespace CatalogueClient
+{
+    class Program
+    {
+        static void Main(string id = null, FileInfo file = null)
+        {
+            if (id != null)
+            {
+                var catalogue = GetCatalogue();
+                var options = GetSerialiserOptions();
+                var work = catalogue.GetWork(id).Result;
+                Console.Write(JsonSerializer.Serialize(work, options));
+                Console.WriteLine();
+            }
+
+            if (file != null && file.Exists)
+            {
+                var catalogue = GetCatalogue();
+                var options = GetSerialiserOptions();
+                var lines = File.ReadAllLines(file.FullName);
+                int counter = 1;
+                foreach (var line in lines)
+                {
+                    var identifier = line.Trim();
+                    if (identifier.IsNullOrEmpty())
+                    {
+                        continue;
+                    }
+                    var work = catalogue.GetWork(identifier).Result;
+                    Console.WriteLine();
+                    Console.WriteLine($"~~~~ start {counter} ~~~~~~~~~~~~~~~~~~~~~~~~~");
+                    Console.Write(JsonSerializer.Serialize(work, options));
+                    Console.WriteLine();
+                    Console.WriteLine($"~~~~ end {counter++} ~~~~~~~~~~~~~~~~~~~~~~~~~~~");
+                    Console.WriteLine();
+                }
+            }
+        }
+
+        private static ICatalogue GetCatalogue()
+        {
+            var logger = new NullLogger<WellcomeCollectionCatalogue>();
```

Comment:
Would it ever be useful to get 'real' logs from `WellcomeCollectionCatalogue` output to console?

Reply:
Yes definitely, I just didn't need one in this little demo